### DISCUSSION TITLE
River community?

### DISCUSSION THREAD

**Discussion by kristopher-wood** [online-ml/river#659] [Q&A] [answered]
Hi all!

My apologies if I missed this somewhere in the documentation or this discussion page, but...

Does River have an online community anywhere for real-time communication? Perhaps a Slack or Discord? Or is this discussion page the only place to talk with other people using it?

Thanks,
Kris

**Comment by MaxHalford** [comment, marked as answer]
Hello there! Nope there is no real time communication tool. But you can just use GitHub discussions, it should be fine and gives more visibility to each discussion.

**Reply by kristopher-wood** [reply]
Cool, thanks for the quick reply! I just wanted to make sure I wasn't missing anything. :)
